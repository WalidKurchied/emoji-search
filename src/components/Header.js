import React, { Component } from 'react';

export default class Header extends Component {
    render() {
        return (
            <header id="header">
                <img
                    src="//cdn.jsdelivr.net/emojione/assets/png/1f638.png"
                    width="32"
                    height="32"
                    alt="header emoji one"
                    className="header-emoji"
                />
                
                <h1 id="emojiAppTitle">Emoji Search</h1>

                <img
                    src="//cdn.jsdelivr.net/emojione/assets/png/1f63a.png"
                    width="32"
                    height="32"
                    alt="header emoji two"
                    className="header-emoji"
                />
            </header>
        );
    }
}