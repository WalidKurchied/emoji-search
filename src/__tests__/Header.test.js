import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../components/Header';

describe('<Header />', () => {
    let component = null; 

    beforeAll(() => {
        component = create(<Header />)
    });

    it ('Should match snapshot', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});