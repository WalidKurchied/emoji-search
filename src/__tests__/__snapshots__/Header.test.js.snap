// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should match snapshot 1`] = `
<header
  id="header"
>
  <img
    alt="header emoji one"
    className="header-emoji"
    height="32"
    src="//cdn.jsdelivr.net/emojione/assets/png/1f638.png"
    width="32"
  />
  <h1
    id="emojiAppTitle"
  >
    Emoji Search
  </h1>
  <img
    alt="header emoji two"
    className="header-emoji"
    height="32"
    src="//cdn.jsdelivr.net/emojione/assets/png/1f63a.png"
    width="32"
  />
</header>
`;
